src/components/App.tsx

```diff
@@ -1,13 +1,14 @@
 import React from 'react';
 import { Switch, Route } from 'react-router-dom';
 import { ValidatorForm, FormState } from './ValidatorForm';
-import { Results, ResultsState, RESULTS_PATH } from './Results';
+import { Results, RESULTS_PATH } from './Results';
+import { ValidationResult } from '../models/HL7Validator';
```

Comment:
Since we now set `baseUrl` in `tsconfig` I think it would be good to adopt some conventions on the use of absolute vs relative imports.  [See here](https://github.com/mcode/pathways/wiki/Code-Style-Guide#module-imports).

Generally if it's in the current or a child directory, use a relative import.  Otherwise use an absolute import.

```suggestion
import { ValidationResult } from 'models/HL7Validator';
```

Reply:
Alright, I'll change that